paymaster/pnpm-lock.yaml

```diff
@@ -1,12 +1,19 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
Are those two things needed? (& why?)

Reply:
hmmm not sure why it's added, I did a `pnpm install` and it's added to this lock file

Reply:
Do you have a .npmrc in your repo?

Reply:
Nope. Anyways, I tried removing them and they still work, so let's remove them I guess.

Reply:
Nope. So I realized these three lines are already in the lock file all the while, it's just moved to the top of the file (previously at the bottom). What it does is to automatically install the dependencies of our dependencies